app/assets/stylesheets/application.scss

```diff
@@ -130,15 +144,44 @@ h1 {
   text-align: center;
 }
 
+
+.searchbra {
+  margin-left: 0;
+}
 .button {
   display: block;
+  padding: 10px;
   margin: 0 auto;
+  color: #000000;
+}
+
+.button:hover {
+  background-color: #6FDF52;
+  color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.